MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift

```
@@ -305,6 +305,7 @@ public class WalletApplicationService: Assertable {
         if success {
             do {
                 try removePaperWallet()
+                try notifySafeCreated()
```

the rest try's work with DB. This one works with network. Are you sure it will be ok in edge case when we will need to retry notification?

---

I would make it required for now, as error handling is another big topic.

---

In removePaperWallet(): let paperWallet = ownerAddress(of: .paperWallet)!
Just move this "try notifySafeCreated()" one line above not to crash app in retry case.

---

welll... I could just `try?` and ignore the error for now